### PR TITLE
Change color of spinner to match themes

### PR DESCRIPTION
**Pull request by Primdahl26**
Changes the spinner color to match the newly implemented themes.

### CODE REVIEW

**Comment by Primdahl26** [comment]
image

to 

image

File: frontend/src/components/loading/spinner.svelte [hunk 1]
```diff
@@ -10,7 +10,7 @@
 </script>
 
 <div class="flex justify-center items-center">
-  <div class="animate-spin rounded-full h-32 w-32 border-b-8 border-neutral" />
+  <div class="animate-spin rounded-full h-32 w-32 border-b-8 border-primary" />
```
Comment (AndreasPB) [inline frontend/src/components/loading/spinner.svelte:13]: Not sure if primary should be used for everything.
Maybe a `state` color makes sense for this
image
(with luxury)

Comment (Primdahl26) [reply frontend/src/components/loading/spinner.svelte:13]: In this specific case i feel like `primary` is what matches all the themes. 

image

image

image

image

image

I have tried a lot of combinations, but this is what looks best IMO.

Comment (AndreasPB) [reply frontend/src/components/loading/spinner.svelte:13]: True, it does looks really nice... I'll find somewhere else to use colors for hints/warnings/errors - Maybe the error page? :D

Comment (Primdahl26) [reply frontend/src/components/loading/spinner.svelte:13]: That could be a good idea 👀

**Review by AndreasPB — APPROVED** [review]

**Pull request merged**